lib/MusicBrainz/Server/Edit/Historic/AddDiscID.pm

```diff
@@ -8,7 +8,7 @@ use MusicBrainz::Server::Translation qw( N_l );
 
 use MusicBrainz::Server::Edit::Historic::Base;
 
-sub edit_name     { N_l('Add disc ID') }
```

Comment:
This should be left as it was, since "disc ID" is an established term and is only relevant to compact discs, not all mediums: http://wiki.musicbrainz.org/Disc_ID